server.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
This python file (server.py) in project root is for example purpose and not used by SDK.
Maybe is better to move this file (and section from README.md related to it) to docs folder?

Reply:
Yea, I agree. It should be in a dedicated `tests` folder or `docs`. I will move it now. Thanks for your feedback! :)